src/tests/ftest/util/file_count_test_base.py

```diff
@@ -90,13 +92,15 @@ def run_file_count(self):
                 self.container = self.add_containers(oclass)
                 self.update_ior_cmd_with_pool(False)
                 try:
+                    self.processes = ior_np
+                    self.ppn = ior_ppn
                     if self.ior_cmd.api.value == 'HDF5-VOL':
                         self.ior_cmd.api.update('HDF5')
-                        self.run_ior_with_flags(
-                            ior_np, ior_ppn, create_pool=False, plugin_path=hdf5_plugin_path,
-                            mount_dir=mount_dir)
+                        self.run_ior_with_pool(
+                            create_pool=False, plugin_path=hdf5_plugin_path, mount_dir=mount_dir)
                     else:
-                        self.run_ior_with_flags(ior_np, ior_ppn, create_pool=False)
+                        self.processes = ior_np
+                        self.run_ior_with_pool(create_pool=False)
```

Comment:
Did you mean to do `self.processes = ior_np` on both lines 95 and 102?

Reply:
good catch... copy/paste error 🐼